File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_ANDROID/SAMPLE_ANDROID/payapp_android_sample_v3/OApi/src/main/java/com/udid/oapi/data/PABillRegisterResponseData.java
package com.udid.oapi.data;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;


/**
 * Create by 김진원
 */
@Getter
@Setter
@ToString(callSuper = true)
@JsonIgnoreProperties(ignoreUnknown = true)
public class PABillRegisterResponseData extends PAResponseData {

    /**
     * 등록결제 연동키
     */
    String encBill;

    /**
     * 등록결제 키
     */
    String billAuthNo;

    /**
     * 카드번호
     */
    String cardno;

    /**
     * 카드사명
     */
    String cardname;

}

File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_IOS/SAMPLE_IOS/payapp_ios_sample_v3/OAPI/model/data/PABillRegisterResponseData.swift
//
//  PABillRegisterResponseData.swift
//  Payapp_iOS_Sample_V2
//
//  Created by 조상현 on 2022/09/26.
//

import ObjectMapper

class PABillRegisterResponseData: PAResponseData {

    
    /**
     * 결제회원 연동키
     */
    var encBill:String?

    /**
     * 결제회원 키
     */
    var billAuthNo:String?
    
    /**
     * 카드번호
     */
    var cardno:String?
    
    /**
     * 카드사명
     */
    var cardname:String?
    
    required convenience init?(map: Map){
        self.init()
    }
    
    override init(){
        super.init()
    }
    
    override func mapping(map: Map) {
        super.mapping(map: map)
        encBill                     <- map["encBill"]
        billAuthNo                  <- map["billAuthNo"]
        cardno                      <- map["cardno"]
        cardname                    <- map["cardname"]
    }
    
}
